### PR TITLE
ncm-spma: add support for more yum.conf options

### LINKED ISSUE #650: ncm-spma: add installonly_limit in yum.conf
For use with el7.2:
The initramfs files are now significantly bigger than in CentOS-7
(1503). You may want to consider lowering installonly_limit in
/etc/yum.conf to reduce the number of installed kernels if your /boot
partition is smaller than 4GB. New installations should consider
using 1GB as the size of the /boot partition.

### PR DESCRIPTION
**Pull request by stdweird**
Fixes #650

### CODE REVIEW

File: ncm-spma/src/main/pan/components/spma/yum/schema.pan [hunk 1]
```diff
@@ -33,8 +33,21 @@ type spma_yum_plugins = {
...
+    "keepcache" ? boolean
+    "retries" ? long(0..) = 10
+    "timeout" ? long(0..) = 30
+};
+
 type component_spma_yum = {
     "userpkgs_retry" : boolean = true
     "fullsearch" : boolean = false
     "plugins" ? spma_yum_plugins
+    "yumconf" ? spma_yum_conf
```
Comment (jrha) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: Is there not a more descriptive name than `yumconf`?

Comment (stdweird) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: suggestions welcome. the way the schema is buildup, i'm reluctant to use something more generic.

Comment (jrha) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: I can't think of anything better that isn't very generic (`main`, `core`, etc.)

Comment (jrha) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: If we just called it `yum`, then the type could be `spma_yum_yum` :yum:

Comment (jrha) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: Can we call it `main_options`? It's what the man pages refer to these options as which makes it usefully googlable.

Comment (stdweird) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: the schema is shared with `yum` and `ips` (and `yumng`), and the options are not generic enough imho. but i'll change it to whatever is most suitable ;)

Comment (jrha) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: Eugh.

Comment (jrha) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: Call it `main_options` and I'll open an issue to have a think about how to improve the way the schema is built without breaking things.

Comment (stdweird) [inline ncm-spma/src/main/pan/components/spma/yum/schema.pan]: done

**Pull request merged**